highest_id.json

```
@@ -0,0 +1 @@
+{"highest_id":6}
```

コミット前のセルフレビューを習慣づけて、そもそもコミットすべきでないファイルをコミットしないようにしましょう。
https://blog.jnito.com/entry/2021/01/30/172607

あと、このファイルはハッシュで保持せずに、単なる数値で保持した方がシンプルになりそうです。

```suggestion
6
```

---

git diffを使えばshow.erbのコメントアウトも気づけたかもしれません。必ず行うようにします！また、コミットに含める・含めないの操作についても今一度確認します。

気が付かなかったです！とてもシンプルになりました！